IRremoteESP8266.cpp

```diff
@@ -33,85 +33,99 @@
-  Serial.print(": ");
-  Serial.print(TICKS_LOW(desired_us + MARK_EXCESS), DEC);
-  Serial.print(" <= ");
-  Serial.print(measured_ticks, DEC);
-  Serial.print(" <= ");
-  Serial.println(TICKS_HIGH(desired_us + MARK_EXCESS), DEC);
-  return measured_ticks >= TICKS_LOW(desired_us + MARK_EXCESS) && measured_ticks <= TICKS_HIGH(desired_us + MARK_EXCESS);
+
+bool  MATCH_MARK(int measured_ticks, int desired) {
+  bool  ret = MATCH_MARK_NOLOG(measured_ticks,desired); 
+  if (!ret) { 
+    Serial.print("MARK NOT Match (");
+    Serial.print(TICKS_LOW(desired ), DEC);
+    Serial.print(" <= ");
+    Serial.print(measured_ticks, DEC);
+    Serial.print(" <= ");
+    Serial.print(TICKS_HIGH(desired), DEC);
+    Serial.print(") Expected : ");
+    Serial.println(desired, DEC);
+  }
+  return ret;
 }
 
-int MATCH_SPACE(int measured_ticks, int desired_us) {
-  Serial.print("Testing space ");
-  Serial.print(measured_ticks * USECPERTICK, DEC);
-  Serial.print(" vs ");
-  Serial.print(desired_us, DEC);
-  Serial.print(": ");
-  Serial.print(TICKS_LOW(desired_us - MARK_EXCESS), DEC);
-  Serial.print(" <= ");
-  Serial.print(measured_ticks, DEC);
-  Serial.print(" <= ");
-  Serial.println(TICKS_HIGH(desired_us - MARK_EXCESS), DEC);
-  return measured_ticks >= TICKS_LOW(desired_us - MARK_EXCESS) && measured_ticks <= TICKS_HIGH(desired_us - MARK_EXCESS);
+bool  MATCH_SPACE(int measured_ticks, int desired) {
+  bool   ret = MATCH_SPACE_NOLOG(measured_ticks,desired);
+  if (!ret) { 
+    Serial.print("SPACE NOT Match (");
+    Serial.print(TICKS_LOW(desired ), DEC);
+    Serial.print(" <= ");
+    Serial.print(measured_ticks, DEC);
+    Serial.print(" <= ");
+    Serial.print(TICKS_HIGH(desired ), DEC);
+    Serial.print(") Expected : ");
+    Serial.println(desired, DEC);
+  }
+  return ret;
 }
 #else
-int MATCH(int measured, int desired) {return measured >= TICKS_LOW(desired) && measured <= TICKS_HIGH(desired);}
-int MATCH_MARK(int measured_ticks, int desired_us) {return MATCH(measured_ticks, (desired_us + MARK_EXCESS));}
-int MATCH_SPACE(int measured_ticks, int desired_us) {return MATCH(measured_ticks, (desired_us - MARK_EXCESS));}
-// Debugging versions are in IRremote.cpp
+  bool MATCH(int measured, int desired) {return measured >= TICKS_LOW(desired) && measured <= TICKS_HIGH(desired);}
+  bool MATCH_MARK(int measured_ticks, int desired) {return MATCH_MARK_NOLOG(measured_ticks, desired );}
+  bool MATCH_SPACE(int measured_ticks, int desired) {return MATCH_SPACE_NOLOG(measured_ticks, desired );}
 #endif
 
 // IRsend -----------------------------------------------------------------------------------
 
 IRsend::IRsend(int IRsendPin)
 {
-	IRpin = IRsendPin;
+  IRpin = IRsendPin;
 }
-
 void IRsend::begin()
 {
-	pinMode(IRpin, OUTPUT);
+  pinMode(IRpin, OUTPUT);
 }
 
-void IRsend::sendCOOLIX(unsigned long data, int nbits)
-{
+void IRsend::sendCOOLIX(unsigned long data, int nbits)  {
   enableIROut(38);
-  mark(COOLIX_HDR_MARK);
+  mark (COOLIX_HDR_MARK);
```

Comment:
Why space here while no space at `space()` in the next line? I would prefer the original `mark(COOLIX_HDR_MARK);` with no space.